boskos/janitor/janitor.go

```diff
@@ -26,15 +28,39 @@ import (
 )
 
 var (
-	clean      = janitorClean
-	POOLSIZE   = 10 // Maximum concurrent janitor goroutines
-	BUFFERSIZE = 1  // Maximum holding resources
+	clean          = janitorClean
+	POOLSIZE       = 10 // Maximum concurrent janitor goroutines
+	BUFFERSIZE     = 1  // Maximum holding resources
+	serviceAccount = flag.String("service-account", "/etc/service-account/service-account.json", "Path to projects service account")
```

Comment:
The deployment.yaml might be less confusing if you explicitly pass --service-account into the container args rather than relying on a magic path default.